data/campaigns/Heir_To_The_Throne/overworld/00a_western_wesnoth.cfg

```diff
@@ -0,0 +1,1625 @@
+#                                                             HALSTEAD, DELWYN, AND DALLBEN
+#######################################################################################################################################################
+#############################
+# HALSTEAD
+#############################
+[event]
+    name=enter hex
+    {FILTER( side=1 {FILTER_LOCATION( x,y,radius=18,31,3 )} )}
+    {FILTER_CONDITION({HAVE_UNIT id,side,search_recall_list=Harper,1,yes})}
+
+    [cancel_action]
+    [/cancel_action]
+    {MOVE_UNDER_KONRAD(
+        {RECALL_XY Harper $stored_konrad.x $stored_konrad.y}
+        {MOVE_UNIT id=Harper "$($stored_konrad.x-1)" $stored_konrad.y}
+        {MODIFY_UNIT id=Harper facing se}
+    )}
+    [message]
+        speaker=Harper
+        message=_"There it be — what’s left of Halstead. You woulda been barely a year or two old, Konrad. It were a nasty fight, but in the end we knocked the whole thing down, Baldras and I did. I miss him something fierce."
+    [/message]
+    [message]
+        speaker=Konrad
+        message=_"Baldras was your uncle, is that right? Is he not back with the others on the Three Sisters?"
+    [/message]
+    [message]
+        speaker=Harper
+        message=_"No. He an’ some o’ the family went off to try and rescue a bunch of slaves, somewhere out eastways. Do a good deed, an’ maybe make us some new friends too, you know? But it’s been years, and not one of thems that went out has come back."
+    [/message]
+    [message]
+        speaker=Harper
+        message=_"Delurin says they’re probably all dead, but I refuse t’ believe it. My uncle Baldras is tougher than that..."
+    [/message]
+    {MOVE_UNDER_KONRAD( {MOVE_UNIT id=Harper $stored_konrad.x $stored_konrad.y} {PUT_TO_RECALL_LIST id=Harper} )}
+[/event]
+#############################
+# DELWYN AND DALLBEN
+#############################
+[event]
+    name=enter hex
+    {FILTER( side=1 {FILTER_LOCATION( x,y,radius=21,19,1 )} )}
+    {FILTER_CONDITION({HAVE_UNIT id,side,search_recall_list=Harper,1,yes})}
+    {FILTER_CONDITION({VARIABLE_CONDITIONAL bm_visited_dallben not_equals yes})}
+
+    [cancel_action]
+    [/cancel_action]
+    {MOVE_UNDER_KONRAD(
+        {RECALL_XY Harper $stored_konrad.x $stored_konrad.y}
+        {MOVE_UNIT id=Harper "$($stored_konrad.x-1)" $stored_konrad.y}
+        {MODIFY_UNIT id=Harper facing se}
+    )}
+    [message]
+        speaker=Harper
+        message=_"Dallben, my home. Might look like jus’ another ruined town to you, but this place was once full of life an’ adventure!"
+    [/message]
+    [message]
+        speaker=Harper
+        message=_"’course everythin’ fell apart come Asheviere. We had to run fer our lives an’ hide, an’ now all’s left is memories. Lotsa stories like ours these days."
+    [/message]
+    [message]
+        speaker=Harper
+        message=_"Wait up a moment, Konrad. I might still be able to scrounge up a li’l gold for us."
```

Comment:
I recognise Harper's speech is supposed to be unrefined, but 'wait up' sounds too modern to me. Prefer 'wait a moment', but I leave it up to you.